From e47772cedeebbe1a0881618cd0fafba72fb4ddef Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Fri, 30 Nov 2018 16:21:40 +0900
Subject: [PATCH 338/909] drm: rcar-du: Add hot plug event function at resume

commit be68c3bc5c79c8f34c514210691447bb23896703 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Because the hotplug interrupt is not executed at resume when
using ADV7511 encoder driver, add processing to be executed hotplug
by the rcar-du driver.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_drv.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_drv.c b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
index e4999fd86f9a..210771214031 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_drv.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
@@ -480,6 +480,9 @@ static int rcar_du_pm_resume(struct device *dev)
 	drm_atomic_helper_resume(rcdu->ddev, rcdu->suspend_state);
 	drm_fbdev_cma_set_suspend_unlocked(rcdu->fbdev, false);
 	drm_kms_helper_poll_enable(rcdu->ddev);
+#if IS_ENABLED(CONFIG_DRM_I2C_ADV7511)
+	drm_helper_hpd_irq_event(rcdu->ddev);
+#endif
 
 	return 0;
 }
-- 
2.17.1

